### ISSUE TITLE
Uncaught TypeError: Cannot read property 'repo' of null

### ISSUE THREAD

**Issue by gjuro87** [akalajzi/atom-file-bookmark#13]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.10.0
**Electron Version**: 0.37.8
**System**: Mac OS X 10.11.6
**Thrown From**: [file-bookmark](https://github.com/akalajzi/atom-file-bookmark) package, v1.0.2
### Stack Trace

Uncaught TypeError: Cannot read property 'repo' of null

```
At /Users/jurica/.atom/packages/file-bookmark/lib/file-bookmark.coffee:162

TypeError: Cannot read property 'repo' of null
    at FileBookmark.loadGitModifiedFiles (/Users/jurica/.atom/packages/file-bookmark/lib/file-bookmark.coffee:162:23)
    at FileBookmark.loadGitModifiedFiles (/Users/jurica/.atom/packages/file-bookmark/lib/file-bookmark.coffee:1:1)
    at HTMLButtonElement.<anonymous> (/Users/jurica/.atom/packages/file-bookmark/lib/file-bookmark.coffee:91:69)
    at HTMLDivElement.jQuery.event.dispatch (/Users/jurica/.atom/packages/file-bookmark/node_modules/jquery/dist/jquery.js:5110:27)
    at HTMLDivElement.elemData.handle (/Users/jurica/.atom/packages/file-bookmark/node_modules/jquery/dist/jquery.js:4918:28)
```
### Commands

```
  2x -7:26.5.0 core:save (atom-text-editor.editor.is-focused)
     -3:45 core:delete (atom-text-editor.editor.is-focused)
  2x -2:50.3.0 core:save (atom-text-editor.editor.is-focused)
     -2:23.1.0 core:copy (atom-text-editor.editor.is-focused)
  4x -2:16.8.0 editor:newline (atom-text-editor.editor.is-focused)
     -2:13.2.0 core:paste (atom-text-editor.editor.is-focused)
  2x -2:11.9.0 editor:newline (atom-text-editor.editor.is-focused)
     -1:57.2.0 core:copy (atom-text-editor.editor.is-focused)
     -1:51.5.0 core:paste (atom-text-editor.editor.is-focused)
     -0:49.2.0 core:delete (atom-text-editor.editor.is-focused)
     -0:33.3.0 core:save (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "customFileTypes": {
      "source.ini": [
        ".buckconfig",
        ".flowconfig"
      ],
      "source.json": [
        ".arcconfig",
        "BUCK.autodeps"
      ],
      "source.python": [
        "BUCK"
      ]
    },
    "disabledPackages": [
      "coffee-lint",
      "terminal-plus",
      "atom-terminal",
      "metrics",
      "tree-view",
      "nuclide"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
atom-beautify, v0.29.11 (active)
atom-terminal, v0.8.0 (inactive)
auto-detect-indentation, v1.2.0 (active)
autocomplete-paths, v1.0.2 (active)
file-bookmark, v1.0.2 (active)
git-diff-details, v1.3.0 (active)
haunter-syntax, v0.3.3 (inactive)
haunter-ui, v0.2.2 (inactive)
language-jade, v0.7.1 (active)
language-soy, v0.3.2 (active)
line-diff-details, v1.4.0 (active)
linter, v1.11.16 (active)
linter-coffeelint, v1.1.2 (active)
monokai, v0.18.0 (inactive)
native-ui, v0.18.3 (inactive)
obscurity-syntax, v0.5.5 (inactive)
princess-syntax, v0.1.3 (inactive)
react, v0.16.0 (active)
set-syntax, v0.3.0 (active)
seti-icons, v1.3.2 (active)
seti-syntax, v1.0.1 (inactive)
seti-ui, v1.3.2 (inactive)
spooki, v0.5.0 (inactive)
term3, v0.22.1 (active)
terminal-plus, v0.14.5 (inactive)
tool-bar, v1.0.1 (active)
unity-ui, v2.1.5 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.1.0 (inactive)
base16-tomorrow-light-theme, v1.1.1 (inactive)
one-dark-ui, v1.5.0 (active)
one-light-ui, v1.5.0 (inactive)
one-dark-syntax, v1.3.0 (active)
one-light-syntax, v1.3.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.5.3 (active)
archive-view, v0.61.1 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.11.2 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.41.0 (active)
bracket-matcher, v0.82.1 (active)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.39.0 (active)
find-and-replace, v0.201.0 (inactive)
fuzzy-finder, v1.3.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.1 (active)
image-view, v0.58.2 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (active)
metrics, v0.53.1 (inactive)
notifications, v0.65.0 (active)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.241.2 (active)
snippets, v1.0.2 (active)
spell-check, v0.67.1 (active)
status-bar, v1.4.1 (active)
styleguide, v0.47.0 (active)
symbols-view, v0.113.0 (inactive)
tabs, v0.100.2 (active)
timecop, v0.33.2 (active)
tree-view, v0.208.2 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.34.0 (active)
whitespace, v0.33.0 (active)
wrap-guide, v0.38.1 (active)
language-c, v0.52.1 (active)
language-clojure, v0.21.0 (active)
language-coffee-script, v0.47.2 (active)
language-csharp, v0.12.1 (active)
language-css, v0.37.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.42.1 (active)
language-html, v0.45.1 (active)
language-hyperlink, v0.16.0 (active)
language-java, v0.23.0 (active)
language-javascript, v0.119.0 (active)
language-json, v0.18.2 (active)
language-less, v0.29.5 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.35.0 (active)
language-php, v0.37.2 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.0 (active)
language-ruby, v0.69.0 (active)
language-ruby-on-rails, v0.25.0 (active)
language-sass, v0.56.0 (active)
language-shellscript, v0.22.4 (active)
language-source, v0.9.0 (active)
language-sql, v0.23.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.28.0 (active)
language-toml, v0.18.0 (active)
language-xml, v0.34.9 (active)
language-yaml, v0.26.0 (active)

# Dev
No dev packages
```

**Comment by eksortso** [comment]
This is affecting me as well.

[Enter steps to reproduce:]

1. Edit files in git repositories.
2. Open the Bookmarked files sidebar.
3. Click "Add all git modified files" button.
4. The error message appears. But some of the files are appearing as expected.

**Atom**: 1.27.1 x64
**Electron**: 1.7.15
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [file-bookmark](https://github.com/akalajzi/atom-file-bookmark) package 1.0.3


### Stack Trace

Uncaught TypeError: Cannot read property 'repo' of null

```
At C:\Users\Dave Ostrroske\.atom\packages\file-bookmark\lib\file-bookmark.coffee:162

TypeError: Cannot read property 'repo' of null
    at FileBookmark.loadGitModifiedFiles (/packages/file-bookmark/lib/file-bookmark.coffee:162:23)
    at FileBookmark.loadGitModifiedFiles (/packages/file-bookmark/lib/file-bookmark.coffee:1:1)
    at /packages/file-bookmark/lib/file-bookmark.coffee:91:69)
    at HTMLDivElement.dispatch (/packages/file-bookmark/node_modules/jquery/dist/jquery.js:5206:27)
    at HTMLDivElement.elemData.handle (/packages/file-bookmark/node_modules/jquery/dist/jquery.js:5014:28)
```

### Commands

```
     -4:31.8.0 github:toggle-git-tab (input.hidden-input)
     -3:22.7.0 core:save (input.hidden-input)
     -3:06.8.0 core:copy (input.hidden-input)
     -3:05.3.0 core:paste (input.hidden-input)
     -3:03.1.0 editor:select-to-beginning-of-word (input.hidden-input)
     -3:02.6.0 core:select-right (input.hidden-input)
  2x -3:02.0 core:delete (input.hidden-input)
     -2:59.7.0 core:move-to-bottom (input.hidden-input)
     -1:32.9.0 core:save (input.hidden-input)
     -1:25 core:close (input.hidden-input)
     -1:19 file-bookmark:toggle-panel (input.hidden-input)
     -1:07.4.0 core:save (input.hidden-input)
     -1:03.6.0 git-blame:toggle (input.hidden-input)
     -1:01.4.0 core:save (input.hidden-input)
```

### Non-Core Packages

```
atom-beautify 0.32.4 
atom-overtype-mode 0.4.0 
atom-password-generator 0.4.2 
convert-to-utf8 0.2.4 
ctrl-dir-scroll 0.2.5 
docblock-python 0.6.0 
docblockr 0.13.7 
file-bookmark 1.0.3 
fonts 3.0.3 
git-blame 1.6.0 
highlight-selected 0.13.1 
insert-timestamp 0.2.4 
iso8601-timestamp 0.1.1 
jquery-snippets 11.0.0 
language-blade 0.40.0 
language-cshtml 0.4.2 
language-powershell 4.0.1 
language-sql-mysql 0.3.1 
language-tsql 0.19.0 
language-vba 0.2.5 
language-vbscript 0.9.0 
line-length-break 0.4.0 
lipsum 0.1.0 
merge-conflicts 1.4.5 
minimap 4.29.8 
minimap-bookmarks 0.4.2 
minimap-cursorline 0.2.0 
minimap-find-and-replace 4.5.2 
minimap-git-diff 4.3.1 
minimap-highlight-selected 4.6.1 
minimap-pigments 0.2.2 
minimap-selection 4.5.0 
minimap-split-diff 0.3.7 
multi-cursor 2.1.5 
multi-paste 0.3.1 
open-recent 5.0.0 
php-cs-fixer 4.1.1 
platformio-ide-terminal 2.8.1 
pretty-json 2.0.0 
read-only-indicator 0.7.1 
readonly 0.3.0 
simple-align 0.2.0 
sort-lines 0.18.0 
split-diff 1.5.2 
status-bar-blame 2.0.3 
sync-settings 0.8.6 
tabs-to-spaces 1.0.5 
todo-show 2.3.0 
wordcount 2.10.4 
```

**Comment by akalajzi** [comment]
Hopefully fixed with 1.0.4 patch

**Comment by LemonDoodles** [comment]
This is still an issue. v 1.0.4
